nexus_constructor/main_window.py

```diff
@@ -62,6 +67,25 @@ def setupUi(self, main_window):
         self.file_writer_control_window = None
```

Comment:
FileWriter control window should be also registered. Otherwise one can click several times on Open filewriter from the menu and it will open several windows.